### DISCUSSION TITLE
第1节课笔记(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#65] [笔记]
请在下方提交第 1 节课笔记笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by li-chuanwei** [comment]
https://www.yuque.com/epiphany-erkzt/epcy7g/vfc8kknf332nmgxs?singleDoc# 《书生·浦语大模型全链路开源开放体系》
语雀文档，新手小白，还望多多包含。

**Reply by RangeKing** [reply]
最好能加上一些自己的总结哦

**Comment by cookieswolf** [comment]
[01-书生·浦语大模型全链路开源体系](https://www.mubu.com/doc/7kIwllDCcUO )

**Comment by haizeigh** [comment]
有道笔记 https://note.youdao.com/s/aJSrTNs8   
《书生·浦语大模型全链路开源开放体系》 https://www.bilibili.com/video/BV1Rc411b7ns/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f

**Comment by xiaomile** [comment]
[1-书生·浦语大模型全链路开源开放体系](https://blog.csdn.net/weixin_42567071/article/details/135375937)

**Comment by zhangzhidaSunny** [comment]
第一课笔记《书生·浦语大模型全链路开源开放体系》
https://blog.csdn.net/m0_65719612/article/details/135373982

**Comment by wangxuejiao** [comment]
第一课笔记《书生·浦语大模型全链路开源开放体系》
https://zhuanlan.zhihu.com/p/676049881

**Comment by hahahamamama** [comment]
第一课笔记
https://github.com/hahahamamama/InternLM/blob/main/Day1%20Notes.md

**Comment by RangeKing** [comment]
第一课笔记
https://rangeking.github.io/#/page/1.%20internlm%20open-source%20system

**Comment by Hyt560** [comment]
书生·浦语大模型实战营学习笔记（一）https://blog.csdn.net/Hyt_foxxx/article/details/135372433

**Comment by AIzealotwu** [comment]
This is my note of the first day lesson.
https://blog.csdn.net/m0_72806612/article/details/135402487

**Comment by liuxianyi** [comment]
This is my node on Notion for InternLM.
https://liuxianyi.notion.site/a3e062529f64460f9277dafee890979a?pvs=4

**Comment by blissSeven** [comment]
书生·浦语大模型全链路开源开放体系
https://github.com/blissSeven/knowledge/blob/master/LLM/1%E4%B9%A6%E7%94%9F-%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E4%BD%93%E7%B3%BB.md

**Comment by KJ-Huang-sz** [comment]
第一课笔记《书生·浦语大模型全链路开源开放体系》https://www.bilibili.com/read/cv29096418/?jump_opus=1

**Comment by nxznm** [comment]
第一课：https://zhuanlan.zhihu.com/p/676433013

**Comment by GingerNg** [comment]
class1: https://note.youdao.com/s/Dlh2VmHC

**Comment by ShelterWFF** [comment]
class1:  http://t.csdnimg.cn/4XRNi

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/676490748

**Comment by hzz12138** [comment]
class1：https://note.youdao.com/s/OixzxDe7

**Comment by AIzealotwu** [comment]
unsubscribe




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Vince ***@***.***&gt;; 
发送时间: 2024年1月7日(星期天) 下午2:13
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [InternLM/tutorial] 第1节课笔记(9班) (Discussion #65)





 
class1：https://note.youdao.com/s/OixzxDe7

**Comment by nini0919** [comment]
This my note for class 1:
https://morning-gaura-eed.notion.site/01-e9a46d27b19140bf99818d83d4708b0d

**Comment by dragonzzl** [comment]
https://blog.csdn.net/dragon_170/article/details/135440385

**Comment by xingyuan3** [comment]
class1：https://blog.csdn.net/weixin_51231310/article/details/135441128

**Comment by dragonzzl** [comment]
你好，邮件我已收到！

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135438224

**Comment by leonnapsun** [comment]
https://zhuanlan.zhihu.com/p/676517560

**Comment by i-am-so-lucky-today** [comment]
第一课《书生·浦语大模型全链路开源开放体系》上课笔记
https://blog.csdn.net/weixin_49534890/article/details/135442666?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135442666%22%2C%22source%22%3A%22weixin_49534890%22%7D

**Comment by wocclyl** [comment]
第一课《书生·浦语大模型全链路开源开放体系》上课笔记
https://m1ti1x9upsq.feishu.cn/wiki/NxErwSJOLisqdwkrjCEcE4p5nSb?from=from_copylink

**Comment by liweiwp** [comment]
第一课《书生·浦语大模型全链路开源开放体系》笔记
https://zhuanlan.zhihu.com/p/676755346
